prime-router/metadata/schemas/iPatientCare/iPatientCare-covid-19.schema

```diff
@@ -16,6 +16,8 @@ elements:
     - name: processing_mode_code
       csvFields: [{name: processing_mode_code}]
 
+    - name: message_id
```

Comment:
Wow!  This is a bug in the schema.  This message ID needs to be here because the Reddy schema uses it.  Nice catch!!!!!

Reply:
We should write a ticket to write in validation for all fields to have a type.

Reply:
Should I write the ticket?

Reply:
Yes, just write the ticket and put in the description what you found here.